session/bootstrap.go

```diff
@@ -649,6 +658,13 @@ func upgradeToVer23(s Session) {
 	doReentrantDDL(s, "ALTER TABLE mysql.stats_histograms ADD COLUMN `flag` bigint(64) NOT NULL DEFAULT 0", infoschema.ErrColumnExists)
 }
 
+// upgradeToVer24 initializes `Sysstem` timezone according to docs/design/2018-09-10-adding-tz-env.md
+func upgradeToVer24(s Session) {
+	sql := fmt.Sprintf(`INSERT HIGH_PRIORITY INTO %s.%s VALUES ("%s", "%s", "TiDB Global System Timezone.")`,
```

Comment:
Will it cause a write conflict error or duplicated entry?

Reply:
Other upgrade functions use `ON DUPLICATE KEY UPDATE`